odoo/http.py

```diff
@@ -1273,6 +1282,15 @@ def session_store(self):
         # Setup http sessions
         path = odoo.tools.config.session_dir
         _logger.debug('HTTP sessions stored in: %s', path)
+        if ODOO_DISABLE_SESSION_GC:
+            _logger.debug('Default session_gc() is disabled')
+        else:
+            _logger.debug('Default session_gc() is enabled; '
+                          'for high-volume systems, it is recommended to disable it by '
+                          'running odoo with the env variable ODOO_DISABLE_SESSION_GC=1 '
+                          'and to setup instead a system cron job to take care of '
+                          'the session expiration, e.g: '
+                          '@hourly find %s -type f -mtime +7 -delete >/dev/null', path)
```

Comment:
I'd like to find a spot in the documentation where this extra info can be put, to increase the chances that whoever needs it will be able to find it. Here we could just log (INFO?) when the default behavior has been disabled.

@mart-e suggests 2 possible places for the doc:
- adding a new section "Performance" in the [deployment doc](https://www.odoo.com/documentation/14.0/administration/deployment/deploy.html).
- or a section about "Environment variables" in the [CLI doc](https://www.odoo.com/documentation/14.0/developer/reference/cmdline.html)

What do you think?


```suggestion
        if ODOO_DISABLE_SESSION_GC:
            _logger.info('Default session GC disabled, manual GC required.')
```

Reply:
I believe the deployment doc is the best place as it became the go-to reference for people looking to fine-tune their system.